tests/test_integrated_channels/test_integrated_channel/test_exporters/test_content_metadata.py

```diff
@@ -39,8 +39,6 @@ def setUp(self):
             enterprise_customer=self.enterprise_customer_catalog.enterprise_customer,
         )
 
-        # Mocks
```

Comment:
We should leave a `# Mocks` comment since only one line from the mock block is removed.

Reply:
Comment `#Mock` applies only to `self.mock_enterprise_customer_catalogs`

Reply:
Comment `#Mock` applies only to `self.mock_enterprise_customer_catalogs`